src/Components/Shared/ExerciseTitle.tsx

```diff
@@ -1,8 +1,13 @@
-export const ExerciseTitle = () => {
+interface Props {
+  title: string | undefined; // undefined allowed only because useParams does not guarantee a string.
```

Comment:
It guarantees a string if the binding is in the object, but it does not guarantee the binding to exist - the binding is not defined in the object, it is "undefined".